src/abstractdataframe/reshape.jl

```diff
@@ -171,12 +173,19 @@ unstack(df::AbstractDataFrame)
 
 * `value` : the value column, defaults to `:value`
 
+* `renamecols` : the function that takes column names from `colkey` and transforms
```

Comment:
I'd say:
"a function that takes a `Vector{Symbol}` of column names from `colkey` and returns target column names as a vector of `Symbol`s or `String`s"

Reply:
actually now `colkey` can be anything as long as `renamecols` returns a vector of `Symbol`, `AbstractString` or `Vector{UInt8}`. I will update the docs

Reply:
And it works elementwise (I will make this clear also)

Reply:
done. Actually I renamed the kwarg to `renamecol` to make it clear that it operates on a single column name.